doc/latex.rst

```diff
@@ -506,46 +544,40 @@ It defaults to empty.  If non-empty, it will be passed as argument to the
    \usepackage{sphinx}
    \sphinxsetup{key1=value1, key2=value2,...}
 
-.. versionadded:: 1.5
-
-.. hint::
```

Comment:
Why did you remove `hint` directive? The "raw" use case is a tip for `\sphinxsetup` macro in LaTeX level. But this section describes about the `latex_elements['sphinxsetup']` key first. It is confusable.